### ISSUE TITLE
Some problems encountered in preparing data

### ISSUE THREAD

**Issue by ZYCheng777** [weiyithu/SurroundDepth#5]
Hi!
I follow the README to prepare DDAD data. But after I perform sift and match operations, I find that the content contained in the sift and match folders is not complete. There should be folders from 000 to 199, but the sift and match folders are not complete, my sift folder only contains 000 to 106, and my match folder only contains 001 to 144. The same problem also exists in the depth folder, the numbered files in this folder are complete (000 to 199), but some files are empty. Why are these situations?

**Comment by weiyithu** [comment]
Hi, have you ever seen any error message? For depth, we directly use [dgp codebase](https://github.com/TRI-ML/dgp) to generate groundtruth depth and we think there should not exist bugs. Have you check that whether downloaded RGB images are complete?

**Comment by ZYCheng777** [comment]
Hi!
I checked the integrity of the downloaded DDAD dataset and re-prepare the data, but the problem is not resolved. When I run sift_ddad.py only the following content appears, then the run stops automatically, and the generated folder is empty.
![1 35ZAYRF$9W6A`_G1CPTFV](https://user-images.githubusercontent.com/76484372/173768381-e41f4d4e-0171-4166-8aa7-2b4151d2a94d.png)
The content in my sift environment is like this
![W78VRHUV`1~ Q49ZA5EKA7W](https://user-images.githubusercontent.com/76484372/173769016-a5572f0a-58d2-4241-bc97-0ae2c7aa684e.png)

**Comment by weiyithu** [comment]
Hi，does the depth folder also have this problem?

**Comment by weiyithu** [comment]
Oh, for depth we only generate validation set, so 000-149 should be empty. However, for sift, we generate train set,  and there should be 100-149 folders. You can delete multi-processing in sift_ddad.py and see whether there is a bug. Here is my wechat 18600500891, you can feel free to contact me.

**Comment by chinmay1148** [comment]
Facing a similar issue.  There are 200 scenes in ddad.  
Had no issue with  $ python export_gt_depth_ddad.py val
But $python sift_ddad.py is only generating sift data for 139 scene.
And $ python match_ddad.py is generating it for 110 scenes only.
Because of which the unable to train ddad_sfm_pretraining step.

How  to generate data using sift_ddad.py and match_ddad.py ?

**Comment by weiyithu** [comment]
We use multiprocessing when generating sift data https://github.com/weiyithu/SurroundDepth/blob/0156aefe2ae152c539b2656751e1ddb99a74141d/tools/sift_ddad.py#L59 and it my not report bug. You can close the multiprocessing and see where the bug is.
